run_explorer.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -z  "$1" ]; then
+	echo "Usage: run_explorer.sh [COSMOGONY_FILE]"
+	exit 1
+fi
+
+sudo docker-compose up -d
```

Comment:
I don't like having a sudo in a script (and for example I don't need it). it might be better to remove it (and add a small section in the readme telling it might be needed to run the script ?)

Reply:
Yes definitely. That's also a problem if the import lasts for too long, and the next `sudo` is blocked, waiting for a password.

Reply:
I updated the script.